tests/acceptance/features/apiSpaces/tusUpload.feature

```diff
@@ -21,11 +21,13 @@ Feature: upload resources using TUS protocol
 
   Scenario: upload a file bigger than the set quota to a project space
     Given user "Alice" has created a space "Project Jupiter" of type "project" with quota "10"
-    When user "Alice" creates a new TUS resource for the space "Project Jupiter" using the WebDAV API with these headers:
-      | Upload-Length   | 100                       |
+    When user "Alice" creates a new TUS resource for the space "Project Jupiter" with content "file content is 24 bytes" using the WebDAV API with these headers:
+      | Upload-Length   | 24                       |
       # dXBsb2FkLnR4dA== is the base64 encoded value of filename upload.txt
       | Upload-Metadata | filename dXBsb2FkLnR4dA== |
+      | Content-Type | application/offset+octet-stream |
       | Tus-Resumable   | 1.0.0                     |
+      | Tus-Extension | creation-with-upload |
```

Comment:
```suggestion
     | Upload-Length   | 24                              |
      # dXBsb2FkLnR4dA== is the base64 encoded value of filename upload.txt
      | Upload-Metadata | filename dXBsb2FkLnR4dA==       |
      | Content-Type    | application/offset+octet-stream |
      | Tus-Resumable   | 1.0.0                           |
      | Tus-Extension   | creation-with-upload            |
```

Reply:
I ran `make test-php-style-fix`. Why it didn't solve it?

Reply:
i think `make test-php-style-fix` fix the styling of PHP file but the problem is with the feature file table formatter.